.travis.yml

```diff
@@ -41,6 +41,7 @@ before_install:
 
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
+  - php bin/update_hostname_validator.php --check-only
```

Comment:
Put this

`if [[ $EXECUTE_HOSTNAME_CHECK  == "true" && $TRAVIS_PULL_REQUEST == "false" ]]; then bin/update_hostname_validator.php --check-only; fi`